lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -19,120 +20,186 @@
 # WANT_JSON
 # POWERSHELL_COMMON
 
+
+Function CheckModified-File($url, $dest, $headers, $credentials, $timeout, $use_proxy, $proxy) {
```

Comment:
BTW You mis-represented the current implementation which is:
```Powershell
if (-not $use_proxy) {
    # Ignore the system proxy settings
    $webClient.Proxy = $null
} elseif ($proxy) {
    $webClient.Proxy = $proxy
}
```
These are the 3 cases:
1. `if (-not $use_proxy)` --> Ignore the system proxy settings
2. `elseif ($proxy)` --> Use the provided proxy
3. `else` --> Do nothing, i.e. use the system proxy settings (default behaviour today)